src/features/team-format-page.js

```diff
@@ -0,0 +1,132 @@
+    }
+    $('body').append(jquiCSS, jquiJS, teamCSS);
+
+    $("h2").remove();
+    $("#about").prepend($("#banner_header"));
+
+    var followTeamButton = $("<a>", {"id":"followTeamBtn", "data-ember-action":"135", "class":"js-follow follow button primary action"});
+    followTeamButton.text("Follow The Whole Team");
+    followTeamButton.click(function(e) {
+        followTeam();
+    });
+    $("#team_logo").after(followTeamButton);
+
+    var newDiv = $("<div>", {"id":"team_chat"});
+    $(".wrapper.c12.clearfix").append(newDiv);
+
+    $(document).click(function(e) {
+        if(e.target.id =="sharebtn") {
+            var o = $(e.target).offset(),
+                offsetLeft = o.left + 1,
+                offsetTop = o.top + 42;
+            $("#share").css({"top":offsetTop+"px", "left":offsetLeft+"px"}).toggle("blind");
+        }
+    });
+
+    $(window).resize(function() {
+        var vpWidth = $(window).width(),
+            vpHeight = $(window).height(),
+            paddedWidth = vpWidth - 40;
+
+        if(paddedWidth < 985) {
+            paddedWidth = 985;
+        }
+
+        var playerWidth = paddedWidth - 685,
+            arMultiplier = (playerWidth - 2) / 16,
+            playerHeight = (9 * arMultiplier) + 32;
+
+        $("div.main.discovery.team").css("width", vpWidth+"px");
+        $("div.wrapper.c12.clearfix").css("width", paddedWidth+"px");
+        $("#player_column").css("width", playerWidth+"px");
+        $("#site_footer").css("width", playerWidth+"px");
+        $("#standard_holder").css({"width":(playerWidth - 2)+"px", "height":playerHeight+"px"});
+
+        if($("#team_chat").length) {
+            var chatHeight = $("#live_player").height() - 2;
+            $("#team_chat").css("height", chatHeight+"px");
+        }
+
+        var teamListHeight = ( ($("#live_player").height() - $("#team_logo").height()) - 76 );
+        $("#team_member_list").css({"height":teamListHeight+"px"});
+    });
+
+    var maxId = setTimeout(function() {}, 0);
+    for(var i=0; i < maxId; i+=1) { 
+            clearTimeout(i);
+    }
+
+    var followList = "none",
+        uName = cookie.get("login");
+        
+    vars.teamCurrentChannel = ($(".js-playing").attr("id")).replace("channel_", "");
+    vars.teamFirstLoad = 1;
+    setTimeout(loadTeam, 1000);
+    
+    var followTeam = function() {
+        if(typeof uName === "undefined") {
+            alert("You need to log in first!");
+        } else {
+            followList = [];
+            
+            vars.jsnTeam.forEach(function(a) {
+                followList.push(a.channel.name);
+            });
+            throttledFollow();
+        }
+    }
+    
+    var throttledFollow = function() {
+        if(followList.length > 0) {
+            var targetChan = followList[0];
+            $("#followTeamBtn").css({"background-color":"#B9A3E3"});
```

Comment:
You call ``$("#followTeamBtn")`` repeatedly. Save it to a variable like ``$followTeamBtn``, where the ``$`` means it's a jQuery object.